app/src/main/java/hr/foi/air/sportloc/view/ui/fragment/CommentListFragment.java

```diff
@@ -0,0 +1,191 @@
+                String userComment = comment.getComment();
+                String commentatorUsername = comment.getCommentator();
+
+                usernameListArray[i] = commentatorUsername;
+            }
+
+            ArrayAdapter arrayAdapter = new ArrayAdapter(getContext(), android.R.layout.simple_list_item_1, usernameListArray);
+            listView.setAdapter(arrayAdapter);
+
+            listView.setOnItemClickListener(new AdapterView.OnItemClickListener() {
+                @Override
+                public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+                    Object clickItemObj = parent.getAdapter().getItem(position);
+                    String selectedUsername = clickItemObj.toString();
+                    String userComment = "";
+
+                    for (int i = 0; i < list.size(); i++) {
+                        CommentModel comment;
+                        comment = list.get(i);
+                        String commentatorUsername = comment.getCommentator();
+                        if (commentatorUsername.equals(selectedUsername)) {
+                            userComment = comment.getComment();
+                        }
+                    }
+
+                    AlertDialog alertDialog = new AlertDialog.Builder(getContext()).create();
+                    TextView title = new TextView(getContext());
+                    title.setText(getResources().getString(R.string.comment_commentator_username) + clickItemObj.toString());
+
+                    int usernameLength=clickItemObj.toString().length();
+                    SpannableString ss= new SpannableString(title.getText());
+                    ClickableSpan clickableSpan=new ClickableSpan() {
+                        @Override
+                        public void onClick(View widget) {
+                            UserModel user = new UserModel();
+                            user.setUsername(selectedUsername);
+                            Map<String, Object> map = new HashMap<>();
+                            map.put(ModelEnum.UserModel.name(), user);
+                            IntentManager.startActivity(getContext(), ProfileActivity.class, map);
+                        }
+
+                        @Override
+                        public void updateDrawState(TextPaint ds) {
+                            super.updateDrawState(ds);
+                            ds.setColor(Color.BLUE);
+                            ds.setFakeBoldText(true);
+                            ds.setUnderlineText(false);
+                        }
+                    };
+                    usernameLength=usernameLength+20;
+                    ss.setSpan(clickableSpan,20,usernameLength,Spanned.SPAN_EXCLUSIVE_EXCLUSIVE);
+                    title.setText(ss);
+                    title.setMovementMethod(LinkMovementMethod.getInstance());
+
+
+
+                    title.setPadding(10, 10, 10, 10);
```

Comment:
1. usernameLength+=20;
2. I believe that the design related code can be moved to xml
3. formatting...... please....